alacritty_terminal/src/term/mod.rs

```diff
@@ -249,11 +173,27 @@ impl<'a, C> RenderableCellsIter<'a, C> {
         RenderableCellsIter {
             cursor: term.renderable_cursor(config),
             grid,
-            inner,
+            inner: grid.display_iter(),
             selection: selection_range,
             config,
             colors: &term.colors,
+            search: RenderableSearch::new(term),
+        }
+    }
+
+    /// Check if point is part of a search match.
+    fn is_matched(&mut self, point: Point) -> bool {
```

Comment:
The check function shouldn't mutate state under the hood. I'm not expecting function that checks wether x is true, to mutate the state. If anything the name should be updated. The return of `bool` as a side effect is likely 'ok'.

Reply:
What name would you suggest then?